presto-hive/src/main/java/io/prestosql/plugin/hive/HdfsConfig.java

```diff
@@ -42,19 +47,19 @@
     private int fileSystemMaxCacheSize = 1000;
 
     @NotNull
-    public List<String> getResourceConfigFiles()
+    public List<File> getResourceConfigFiles()
```

Comment:
as you're changing this, why not `Path`?

(i'm not sure the change is needed thought)

Reply:
I followed the `File` type of keystorePath & trustStorePath in ElasticsearchConfig. Exactly, we can also use Path. Actually, this type change isn't mandatory, but I assume using the correct type is better than string for clarifying the object meaning.